installer/roles/ansible-service-broker-setup/tasks/main.yml

```diff
@@ -58,7 +58,7 @@
       mode: u+x
 
   - name: Execute OpenShift Ansible Broker (OAB) provision script
-    shell: bash -e /tmp/provision-ansible-service-broker.sh "{{ dockerhub_username }}" "{{ dockerhub_password }}" "{{ dockerhub_org }}" "{{ launch_apb_on_bind }}" "{{dockerhub_tag}}" "{{ wildcard_dns_host }}" "{{ ansible_service_broker_namespace }}"
+    shell: "bash -e /tmp/provision-ansible-service-broker.sh {{ dockerhub_username }} {{ dockerhub_password }} {{ dockerhub_org }} {{ launch_apb_on_bind }} {{dockerhub_tag}} {{ cluster_public_ip }} {{ wildcard_dns_host }} {{ ansible_service_broker_namespace }}"
```

Comment:
@vchepeli looks like it needs to be escaped here too. Not sure why the escaping was removed

Reply:
> Not sure why the escaping was removed

There was no comment why it escapes every symbol and I removed it